test/components/plots/areaPlotTests.ts

```diff
@@ -17,6 +17,24 @@ describe("Plots", () => {
       assert.strictEqual(plot.height(), 400, "was allocated height");
       svg.remove();
     });
+
+    it("adds a padding exception to the y scale at the constant y0 value", () => {
+      var svg = TestMethods.generateSVG(400, 400);
+      var xScale = new Plottable.Scales.Linear();
+      var yScale = new Plottable.Scales.Linear();
+      var constantY0 = 30;
+      yScale.addExtentsProvider((scale: Plottable.Scales.Linear) => [[constantY0, constantY0 + 10]]);
+      yScale.autoDomain();
+      assert.operator(yScale.domain()[0], "<", constantY0, "lower end of y Scale's domain is normally padded");
```

Comment:
I don't really think the above few lines are necessary.  Note that the `plot` is never used before these lines yet this is in "areaPlotTests.ts"